src/main/java/com/univocity/parsers/containers/Csv.java

```diff
@@ -0,0 +1,48 @@
+package com.univocity.parsers.containers;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+/**
+ * @author naveen.kasthuri
+ *
+ * A container that maps column indices to thier names. This avoids having duplicate arrays of
+ * key-value pairs where the keys are the same for every row in the csv file.
+ *
+ * This is used by the Row object and the RowListProcessor. Should not be used otherwise.
+ */
+public class Csv {
+  // Contains a mapping between row index and column
+  private final Map<String, Integer> columnIndexMap;
```

Comment:
I could attempt to make the constructors more flexible such that this class accepts an EnumMap as part of its constructor. Then, the user can construct using that and not have to rely on the existence of column headers in the csv file. Let me see what I can do with this.